packages/vue/src/components/password-text-box.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <dx-text-box
+    :model-value="value"
+    :styling-mode="stylingMode"
+    :placeholder="placeholder"
+    :mode="isPasswordMode ? 'password' : 'text'"
+    value-change-event="keyup input change"
+    @update:model-value="changeValue"
+  >
+    <dxi-button
+      name="today"
+      location="after"
+      :options="{
+        visible: valueState?.length > 0,
+        icon: 'icons/eye.svg',
+        onClick: switchMode
+      }"
+    />
+    <dx-validator
+      :validation-rules="[
+        {type: 'required', message: 'Password is required'}, ...validators]"
+    />
+  </dx-text-box>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue';
+import { DxTextBox, DxButton as DxiButton } from 'devextreme-vue/text-box';
```

Comment:
Why do you rename DxButton?

Reply:
I think its more readable, in that place dxButton is nested for DxTextbox